pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -76,6 +76,11 @@ func (srv PrometheusSrv) RouteGetRuleStatuses(c *models.ReqContext) response.Res
 		return ErrResp(http.StatusInternalServerError, err, "failed to get namespaces visible to the user")
 	}
 
+	if len(namespaceMap) == 0 {
+		srv.log.Debug("User does not have access to any of the namespaces")
```

Comment:
I think it would be good to log the User ID here too?

```suggestion
		srv.log.Debug("User does not have access to any namespaces")
```